src/com/google/enterprise/adaptor/opentext/OpentextAdaptor.java

```diff
@@ -2457,6 +2642,22 @@ public void configure(Config config) {
       } else {
         this.iis = CwsServer.IIS.name().equalsIgnoreCase(server);
       }
+      try {
+        this.readTimeout = Integer.parseInt(
```

Comment:
Why is this stuff done twice? Is there a problem with passing the needed values from init after they are parsed, rather than calling configure(config) at the top of init?